src/pages/starter-kit/checkout/payment-reference.md

```diff
@@ -15,13 +15,169 @@ The checkout module provides REST and GraphQL APIs to configure out-of-process p
+| `title`                   | String  | No       | Display name of the payment method.                                                                                                                                               |
+| `description`             | String  | No       | Description of the payment method.                                                                                                                                                |
+| `active`                  | Boolean | Yes      | Status indicating if the method is active.                                                                                                                                        |
+| `backend_integration_url` | String  | No       | URL for backend integration, which is an App Builder URL.                                                                                                                         |
+| `stores`                  | Array   | No       | List of store codes that payment method is available for.                                                                                                                         |
+| `order_status`            | String  | No       | Initial [order status](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/order-management/orders/order-status) when using this method. Default is `pending`. |
+| `countries`               | Array   | No       | List of countries where the method is available.                                                                                                                                  |
+| `currencies`              | Array   | No       | Currencies supported by the payment method.                                                                                                                                       |
+| `custom_config`           | Array   | No       | Custom configuration settings for payment methods.                                                                                                                                |
+
+<CodeBlock slots="heading, code" repeat="2" languages="bash, json" />
+
+#### Example request
+
+```bash
+curl --request POST \
+--url <ADOBE_COMMERCE_API_URL>/V1/oope_payment_method \
+--header 'Authorization: Bearer <TOKEN>' \
+--header 'Content-Type: application/json' \
+--data '{
+  "payment_method": {
+    "code": "method-1",
+    "title": "Method 1",
+    "description": "Description for Method 1",
+    "active": true,
+    "backend_integration_url": "http://example.com",
+    "stores": ["store-1", "store-2"],
+    "order_status": "processing",
+    "countries": ["ES", "US"],
+    "currencies": ["EUR", "USD"],
+    "custom_config": [
+      {
+        "key1": "value1"
+      }
+    ]
+  }
+}'
+```
+
+#### Example response
```

Comment:
Name these two examples differently to indicate what they are exemplifying.

Reply:
The main title is **### Create a new payment method** and this is a subtitle so is a example request of how to create a new payment method.

Reply:
@jcuerdo these subtitles are turned into labels for buttons on a tabbed interface when used with this line of code:
```
<CodeBlock slots="heading, code" repeat="2" languages="bash, json" />
```
So the tabbed interface will have two tabs named "Example response" which does not tell the user what the difference is.